tests/ManageCourses.Tests/SmokeTests/ApiEndpointTests.cs

```diff
@@ -97,7 +97,40 @@ public async Task EnrichmentPublishTest()
             var result = await apiClient.Enrichment_PublishInstitutionAsync(ucasInstitutionCode);
             result.Should().BeTrue();
         }
+        [Test][Ignore("needs search and compare environment up and running for this test to pass")]
```

Comment:
spin up a S+C API or delete this test

reject

Reply:
No, keep the test, it will come in handy.